pkg/pillar/zedcloud/authen.go

```diff
@@ -384,68 +385,92 @@ func VerifySigningCertChain(ctx *ZedCloudContext, content []byte) ([]byte, error
 	err := proto.Unmarshal(content, sm)
 	if err != nil {
 		log.Errorf("SaveCertsToFile: Unmarshal err %v\n", err)
-		return nil, err
+		return []byte{}, err
```

Comment:
Is there a difference between the nil and zero length byte array that matters for the caller?

Reply:
changed